### ISSUE TITLE
Oauth 2.0 tokens cannot be viewed/used/deleted in 4.4.2

### ISSUE THREAD

**Issue by devjack** [a85/POSTMan-Chrome-Extension#2167]
1. Postman Version: Postman 4.4.2 
2. App (Chrome app or Mac app):  Chrome 51.0.2704.103
3. OS details: OS X / x86-64
4. Is the Interceptor on and enabled in the app: Installed & Enabled
5. Did you encounter this recently, or has this bug always been there: Recently encountered, previously sporadic and now occurs every time.
6. Expected behaviour: When I select a token I should be able to view and delete it (See #2106) as well as usable.
   Note that when selecting my first (and subsequent) token the right hand side panel visible refreshes with the text moving ~20 px to the top left. It 'looks' like the panel refreshes but is unable to read the Oauth token and thus falls back on the 'please select a token' message.
7. Console logs (http://blog.getpostman.com/2014/01/27/enabling-chrome-developer-tools-inside-postman/ for the Chrome App, View->Toggle Dev Tools for the Mac app):
   ![screen shot 2016-07-06 at 11 31 08 am](https://cloud.githubusercontent.com/assets/3516066/16604936/bfa2a7e4-436d-11e6-9c9b-96a33f751e31.png)
8. Screenshots (if applicable)
   <img width="1138" alt="screen shot 2016-07-06 at 11 31 14 am" src="https://cloud.githubusercontent.com/assets/3516066/16604923/aa0f1606-436d-11e6-8d98-f3443b26b86f.png">

**Comment by abhijitkane** [comment]
@devjack Thanks for reporting this. Can you open the console (https://www.getpostman.com/docs/errors), enter the following:

```
'['+_.map(pm.oAuth2Tokens.models, function(m) {return JSON.stringify(m.toJSON())})+']'
```

and send the result to help@getpostman.com? This will help is isolate the root cause of the problem. If the output has any sensitive token data, you can erase that, but make sure the overall structure is maintained.

**Comment by numaanashraf** [comment]
@devjack This should be resolved in the latest versions of the app (v4.7.2+). Feel free to re-open this if you are still facing issues.
